decidim-admin/config/locales/en.yml

```
@@ -1046,6 +1046,10 @@ en:
         statistics: Activity
         user_groups: Groups
         users: Administrators
+        edit_content_block: Edit content block
```

There's a missing key from one of my suggestions and also I suggested to remove `edit_content_block`, so:
```suggestion
        moderated_users: Reported participants
```